packages/manager/modules/pci/src/projects/project/streams/stream/backlogRetention/backlog-retention.module.js

```diff
@@ -0,0 +1,25 @@
+import angular from 'angular';
+import '@uirouter/angularjs';
+import '@ovh-ux/ng-translate-async-loader';
+import '@ovh-ux/ng-ovh-cloud-universe-components';
+import 'angular-translate';
+import 'ovh-api-services';
+
+import component from './backlog-retention.component';
+import routing from './backlog-retention.routing';
+
+const moduleName = 'ovhManagerPciStreamsStreamBacklogRetention';
+
+angular
+  .module(moduleName, [
+    'ngTranslateAsyncLoader',
+    'ngOvhCloudUniverseComponents',
+    'pascalprecht.translate',
+    'ovh-api-services',
+    'ui.router',
+  ])
+  .config(routing)
+  .component('ovhManagerPciStreamsStreamBacklogRetention', component)
+  .run(/* @ngTranslationsInject:json ./translations */);
```

Comment:
A module come with it's own translations. If I want to import the module to use his *service/component/whatever*/**one of his translations**, it will works. 
If translations are injected only in routing, I can't use it before accessing the state.

Reply:
Ok, thanks for the explanation ;)